### PR TITLE
Re-enable stacks create/delete

### PR DESCRIPTION
**Pull request by thesquelched**
Re-enable stack creation and delete, as well as tests.

### CODE REVIEW

**Comment by amock** [comment]
+1

File: lavaclient/api/response.py [hunk 1]
```diff
@@ -388,11 +388,11 @@ def refresh(self):
...
-    # def delete(self):
-    #     """
-    #     Delete this stack.
-    #     """
-    #     return self._client.stacks.delete(self.id)
+    def delete(self):
+        """
+        Delete this stack.
+        """
+        return self._client.stacks.delete(self.id)
```
Comment (jimbobhickville) [inline lavaclient/api/response.py]: Do we need a confirm prompt like in cluster delete?  If we do, you can check the returned stack to see if it's even deletable and just refuse it altogether before they attempt to send it to the API.

Comment (thesquelched) [inline lavaclient/api/response.py]: Good point

Comment (thesquelched) [inline lavaclient/api/response.py]: Note that this is the model's `delete` method, which doesn't need a confirm.  I did add one to the CLI command (along with a `--force` option)

**Comment by jimbobhickville** [comment]
+1

**Pull request merged**